iceberg/checkstyle/checkstyle.xml

```diff
@@ -448,7 +448,7 @@
           <property name="max" value="12"/>
         </module>
         <module name="JavadocMethod"> <!-- Java Style Guide: Where Javadoc is used -->
-            <property name="scope" value="public"/>
+            <property name="accessModifiers" value="public"/>
```

Comment:
After this change, you can use the Intellij 2024 with higher CheckStyle-IDEA plugin version 5.94.1:
![idea2024-Checkstyle-IDEA](https://github.com/user-attachments/assets/529cff01-a135-4e67-8f26-84b800665100)

![idea2024-hive-ice-check](https://github.com/user-attachments/assets/9508e572-11fc-49b4-a61c-0d70fc1e6f7e)